openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/MaintenanceService.java

```diff
@@ -192,4 +192,8 @@ private void disableMaintenanceMode() {
         disableMaintenanceMode();
         return handleMaintenanceStatus();
     }
+    
```

Comment:
@wahyupratomoicsg got error if dont add this?

Reply:
If I run "Build with dependencies" from netbeans, yeah they find this method